pkg/controllers/dynakube/extension/servicename/naming.go

```diff
@@ -0,0 +1,18 @@
+package servicename
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/extension/consts"
+)
+
+func BuildPort() string {
```

Comment:
nitpik: `BuildPortName` - Port is confusing to me.

Reply:
fixed in bd1bb17088eca494a70ffcd6062a7ffa7f69f3a7